Sprint-3/todo-list/script.js

```diff
@@ -1,25 +1,97 @@
+const toDoInput = document.getElementById("inputToDo");
+const addToDoBtn = document.getElementById("addToDo");
+const container = document.getElementById("todo-list");
+const removeAll = document.getElementById("remove-all-completed");
+addToDoBtn.addEventListener("click", () => {
+  addNewTodo(event);
+});
+let todos = [];
+
+// Function to populate the todo list
 function populateTodoList(todos) {
   let list = document.getElementById("todo-list");
-  // Write your code to create todo list elements with completed and delete buttons here, all todos should display inside the "todo-list" element.
+  list.innerHTML = ""; // Clear the list to avoid duplications
+
+  todos.forEach((todo, index) => {
+    const listItem = document.createElement("li");
+    listItem.innerText = todo.task;
+    list.appendChild(listItem);
+
+    // Create 'Complete' button
+    const completeBtn = document.createElement("button");
+    completeBtn.innerText = "complete";
+    const newButton1 = listItem.appendChild(completeBtn);
+    newButton1.className = "newButton";
```

Comment:
1. Why not just set the class directly to `completeBtn`?
1. What is the use of the class `newButton`? I can't find this class in `style.css`.

Reply:
newButton is defined but not used in css, I forgot to remove it.
Thanks